Source/WebCore/dom/Element.cpp

```diff
@@ -1318,6 +1318,8 @@ static HashSet<TreeScope*> collectAncestorTreeScopeAsHashSet(Node& node)
 
 int Element::offsetLeftForBindings()
```

Comment:
Is it correct to do this just for script? Do we know if some internal callers may want skipped content layout as well ?

Reply:
It looks like this is only used internally bu spatial navigation code. Anyway it should be safe to move this code to offsetLeft method, done.